endpoints/decorators.py

```diff
@@ -385,14 +385,14 @@ def wrapper(namespace_name, repo_name, *args, **kwargs):
 
             if mirror is None:
                 abort(
-                    500,
+                    401,
```

Comment:
If a repository is in a mirror state, then it's expected to have a mirror config. If it's not there, then we should assume something went wrong on our side. i.e. the user isn't making the api with the mirror config